src/collective/cover/tiles/collection.py

```diff
@@ -55,10 +55,9 @@ class ICollectionTile(IPersistentCoverTile):
         required=False,
     )
 
-    # FIXME: this field should be named 'count'
-    form.omitted('number_to_show')
-    form.no_omit(IDefaultConfigureForm, 'number_to_show')
-    number_to_show = schema.List(
+    form.omitted('count')
+    form.no_omit(IDefaultConfigureForm, 'count')
+    count = schema.List(
```

Comment:
The schema says List, but you are working like if it was a dictionary.. could you please explain the idea behind this schema change? Why a simple Int don't resolve this problem?

Reply:
I only just kept the model already existing schema in the collection tile, but yes, a simple Int resolve this problem.